src/test/java/LoginPageTest.java

```diff
@@ -24,6 +22,11 @@ public void setUp() {
         driver = WebDriverUtils.createWebDriver();
     }
 
+    @BeforeMethod
```

Comment:
Same here why opening page takes place before each test?
If there is no page change during running tests in the class, then opening page should be done in the method annotated with @BeforeMethod